FlorenceBML/BMLacq811.xml

```diff
@@ -28,6 +35,171 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <idno>Marrassini ms. 6</idno>
                         </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
```

Comment:
It just dawned on me that the description is not quite right, also in the other MS, because the structure should be
msItem 1 = Ethiopic Psalter LIT2701Dawit
   msItem 1.1. = Psalms of David LIT2000Mazmur
   msItem 1.2  = Songs of Prophets
         msItem 1.2.1 = Song 1
         msItem 1.2.2 = Song 2
         etc
   msItem 1.3 = Song of Songs LIT2362Songof
   msItem 1.4 = Anqasa Berhan
   msItem 1.5 = Weddase Maryam

because the Ethiopic Psalter https://betamasaheft.eu/works/LIT2701Dawit/main is always a collection of texts
So these are not independent texts but parts of the Psalter

Easiest is always to use as template other MSS descriptions of Psalter and adjust them, check e.g.
ESdd004
DSEthiop4
etc
So the workflow is: identify the main text, find it on BM, look for the list of MSS and copy )

Reply:
When I understand you right, it is not a big deal to make that change afterwards. I just introduced the new structure. You may see, if I did it properly. Concerning the workflow it was exactly my strategy to take the newly updated BMLacq679 as a model and copy the content with little changes.